nvflare/fuel/utils/component_classes.json

```diff
@@ -1 +1,572 @@
+    ],
+    "CCManager": [
+        "nvflare.app_opt.confidential_computing.cc_manager"
+    ],
+    "FlowerClientApplet": [
+        "nvflare.app_opt.flower.applet"
+    ],
+    "FlowerServerApplet": [
+        "nvflare.app_opt.flower.applet"
+    ],
+    "FlowerClientConnector": [
+        "nvflare.app_opt.flower.connectors.flower_connector"
+    ],
+    "FlowerServerConnector": [
+        "nvflare.app_opt.flower.connectors.flower_connector"
+    ],
+    "HECrossSiteModelEval": [
+        "nvflare.app_opt.he.cross_site_model_eval"
+    ],
+    "HEInTimeAccumulateWeightedAggregator": [
+        "nvflare.app_opt.he.intime_accumulate_model_aggregator"
+    ],
+    "HEModelDecryptor": [
+        "nvflare.app_opt.he.model_decryptor"
+    ],
+    "HEModelEncryptor": [
+        "nvflare.app_opt.he.model_encryptor"
+    ],
+    "HEModelSerializeFilter": [
+        "nvflare.app_opt.he.model_serialize_filter"
+    ],
+    "HEModelShareableGenerator": [
+        "nvflare.app_opt.he.model_shareable_generator"
+    ],
+    "PTClientAPILauncherExecutor": [
+        "nvflare.app_opt.pt.client_api_launcher_executor"
+    ],
+    "PTFedAvgEarlyStopping": [
+        "nvflare.app_opt.pt.fedavg_early_stopping"
+    ],
+    "PTFedOptModelShareableGenerator": [
+        "nvflare.app_opt.pt.fedopt"
+    ],
+    "FedOpt": [
+        "nvflare.app_opt.pt.fedopt_ctl",
+        "nvflare.app_opt.tf.fedopt_ctl"
+    ],
+    "PTFileModelLocator": [
+        "nvflare.app_opt.pt.file_model_locator"
+    ],
+    "PTFileModelPersistor": [
+        "nvflare.app_opt.pt.file_model_persistor"
+    ],
+    "PTInProcessClientAPIExecutor": [
+        "nvflare.app_opt.pt.in_process_client_api_executor"
+    ],
+    "PTMultiProcessExecutor": [
+        "nvflare.app_opt.pt.multi_process_executor"
+    ],
+    "JoblibModelParamPersistor": [
+        "nvflare.app_opt.sklearn.joblib_model_param_persistor"
+    ],
+    "SKLearnExecutor": [
+        "nvflare.app_opt.sklearn.sklearn_executor"
+    ],
+    "TFClientAPILauncherExecutor": [
+        "nvflare.app_opt.tf.client_api_launcher_executor"
+    ],
+    "TFInProcessClientAPIExecutor": [
+        "nvflare.app_opt.tf.in_process_client_api_executor"
+    ],
+    "TFModelPersistor": [
+        "nvflare.app_opt.tf.model_persistor"
+    ],
+    "MLflowReceiver": [
+        "nvflare.app_opt.tracking.mlflow.mlflow_receiver"
+    ],
+    "MLflowWriter": [
+        "nvflare.app_opt.tracking.mlflow.mlflow_writer"
+    ],
+    "TBAnalyticsReceiver": [
+        "nvflare.app_opt.tracking.tb.tb_receiver"
+    ],
+    "TBWriter": [
+        "nvflare.app_opt.tracking.tb.tb_writer"
+    ],
+    "WandBReceiver": [
+        "nvflare.app_opt.tracking.wandb.wandb_receiver"
+    ],
+    "WandBWriter": [
+        "nvflare.app_opt.tracking.wandb.wandb_writer"
+    ],
+    "XGBFedController": [
+        "nvflare.app_opt.xgboost.histogram_based.controller",
+        "nvflare.app_opt.xgboost.histogram_based_v2.fed_controller"
+    ],
```

Comment:
How do users determine between V1 vs V2?

Should we put v2 in front of v1?

Reply:
no need for versioning. The component_classes.json is checked in with the same version of codes.